modules/gitops-monitoring-argo-cd-health-using-prometheus-metrics.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * cicd/gitops/monitoring-argo-cd-instances.adoc
+
+:_content-type: PROCEDURE
+[id="gitops-monitoring-argo-cd-health-using-promethous-metrics_{context}"]
+= Monitoring Argo CD health using Prometheus metrics
+
+You can monitor Argo CD health status by using Prometheus metrics queries.
+
+.Procedure
+
+. On the {product-title} web console, to switch *Developer*'s preview.
+. In *Developer*'s preview, select the namespace on which an Argo CD application is installed, go to *Observe* and then select the Metrics tab.
```

Comment:
In the Developer perspective of the web console, and navigate to the *Observe* --> *Metrics*. Your project should be `openshift-gitops` where your Argo CD application is installed.